qodana.yaml

```diff
@@ -29,3 +29,9 @@ exclude:
   - name: CheckNamespace
     paths:
       - Src\FluentAssertions\Polyfill
+  - name: UnusedMethodReturnValue.Global
```

Comment:
🤔 Isn't it better to exclude everything under `Polyfill`? See https://www.jetbrains.com/help/qodana/qodana-yaml.html#exclude-example

Reply:
We don't care about any warnings in code added by the `Polyfill` library